### PR TITLE
Show output window when migration fails

### LINKED ISSUE #4449: Show output window when migration fails
### Description

Fit-and-finish.

If the migration fails the wizard tells the user to look in the output window. In this case we could automatically open the output window.

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Fixes #4449 and #4450

### CODE REVIEW

**Comment by duncanp-sonar** [comment]
There are two unrelated fixes in the PR (I meant to keep them separate but made a mess of it; they are both simple so I've left them together).

File: src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs [hunk 1]
```diff
@@ -159,7 +159,7 @@ public async Task LearnMoreButtonClicked_ShowInBrowserCalled()
             notification.Actions.LastOrDefault().CommandText.Should().Be(MigrationStrings.MigrationPrompt_LearnMoreButton);
             notification.Actions.LastOrDefault().Action(null);
 
-            browserService.Verify(x => x.Navigate(MigrationStrings.LearnMoreUrl), Times.Once);
+            browserService.Verify(x => x.Navigate(MigrationStrings.Url_LearnMoreUrl), Times.Once);
```
Comment (duncanp-sonar) [inline src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs:162]: I've added a second link to the wizard, so I changed the naming so both resources start with "URL_" so they are next to each other in the resource file.

File: src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs [hunk 2]
```diff
@@ -47,30 +47,40 @@ internal class MigrationWizardController : IMigrationWizardController
...
             var finishedSuccessfully = migrationWizardWindow.ShowModal();
 
             if (finishedSuccessfully != null && finishedSuccessfully.Value)
             {
                 MigrationWizardFinished?.Invoke(this, EventArgs.Empty);
             }
+            else
+            {
+                // Show the output window in event of an unsuccessful migration
+                outputWindowService.Show();
```
Comment (duncanp-sonar) [inline src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs:78]: This is one of the changes: show the output window pane if migration fails.
Fit and finish i.e. nice to have but not essential

File: src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml [hunk 3]
```diff
@@ -85,6 +85,15 @@
...
                 </Grid>
 
+                <Border x:Name="tfvcInfo" BorderThickness="2" BorderBrush="Red" Margin="15,20,15,0" >
+                    <TextBlock TextWrapping="WrapWithOverflow" Padding="10">
+                        <Run>NB: If you are using Tfvc source control, please read the additional information 
+                        </Run>
+                        <Hyperlink NavigateUri="http://any" RequestNavigate="OnNavigateToTfvcHelp"> here</Hyperlink>
+                            <Run> before starting.</Run>
+                    </TextBlock>
+                </Border>
```
Comment (duncanp-sonar) [inline src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml:95]: This is the other change I discussed with Marco: extra text for Tfvc users.

Currently this is hard-coded. I'll add a ticket to make it dynamic, but we might not have time to do it.

**Review by bigfluffycookie — APPROVED** [review]

**Pull request merged**